AdtSampleApp/SampleClientApp/CommandLoop.cs

```diff
@@ -365,18 +366,18 @@ public async Task CommandCreateDigitalTwin(string[] cmd)
                 twin_id = cmd[2];
             string[] args = cmd.Skip(3).ToArray();
 
-            Dictionary<string, object> meta = new Dictionary<string, object>()
+            var meta = new Dictionary<string, object>
```

Comment:
Should we change this to use BasicDigitalTwin instead?

Reply:
changed